README.md

```diff
@@ -1,7 +1,7 @@
 <a name="readme-top"></a>
 
 <div align="center">
-  <h3><b>Space Travelers' Hub</b></h3>
+  <h3><b>Crypto Metrics Webapp</b></h3>
 </div>
 
 <!-- TABLE OF CONTENTS -->
```

Comment:
- [ ] We were requested to acknowledge the author of the original design in the Acknowledgement section of the README file. Let's kindly do it.